src/System.Text.JsonLab.Dynamic/System/Text/Json/Deserializer.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Buffers.Text;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Reflection;
+using System.Reflection.Emit;
+
+namespace System.Text.JsonLab
```

Comment:
It think it's fine that you changed the package name, but I would not change the namespace name, i.e. I would live it as System.Text.Json.

Reply:
I had to rename the namespace, here (https://github.com/dotnet/corefxlab/pull/2265) to avoid conflicts with benchmark tests.

> Renamed System.Text.Json to System.Text.JsonLab for now to avoid naming conflict with the System.Text.Json NuGet package by [YSharp](https://github.com/ysharplanguage/FastJsonParser)

Once we port those tests out of the repo (and the investigation is complete), we can change the namespace back.